core/util/logging.js

```diff
@@ -17,7 +17,7 @@ let Info = () => {};
 let Warn = () => {};
 let Error = () => {};
 
-export function init_logging (level) {
+export function init_logging(level) {
```

Comment:
this doesn't look normalized -- some have spaces between parens and the name, some don't.  I prefer spaces.